src/network/bridge.rs

```diff
@@ -82,6 +85,7 @@ impl driver::NetworkDriver for Bridge<'_> {
             ipam,
             mtu,
             isolate,
+            metric: Some(metric),
```

Comment:
I think this is always going to be some value or `100`, since `parse_option` has a default of `100`. I think if not parse-able we should pass `None` here instead of default `100`.

Reply:
Why? Non-parsable should be an error.

Reply:
No my point was that metric defaults to 100 in parse_option, we should not default to any value instead default should be `None`.

Reply:
That would require significant changes to the option parsing code that I would prefer to avoid for the moment. I'm willing to do it if you insist, though

Reply:
No not necessary and blocker for this PR, we can do it in a new PR. I'll open a PR for that after this gets merged.